nf_core/pipeline-template/nextflow.config

```diff
@@ -82,78 +97,73 @@ profiles {
     docker.runOptions = '-u \$(id -u):\$(id -g)'
   }
   singularity {
-    docker.enabled = false
-    singularity.enabled = true
-    podman.enabled = false
-    shifter.enabled = false
-    charliecloud.enabled = false
+    singularity.enabled    = true
     singularity.autoMounts = true
+    docker.enabled         = false
+    podman.enabled         = false
+    shifter.enabled        = false
+    charliecloud.enabled   = false
   }
   podman {
-    singularity.enabled = false
-    docker.enabled = false
-    podman.enabled = true
-    shifter.enabled = false
-    charliecloud = false
+    podman.enabled       = true
+    singularity.enabled  = false
+    docker.enabled       = false
+    shifter.enabled      = false
+    charliecloud         = false
```

Comment:
```suggestion
    charliecloud.enabled = false
```

Reply:
fixed in https://github.com/nf-core/tools/pull/972/commits/3a2c8158b856712cb78b285b0bbe5c23125bf984

Reply:
Not fixed here yet though, @drpatelh needs to pull in upstream updates. Again.. 🙄 Sorry..

Reply:
Should be able to fix these conflicts on `dsl2_template` in a separate PR now it's all on that branch?